nautobot/core/views/generic.py

```diff
@@ -322,6 +323,12 @@ def get(self, request):
             }
             RequestConfig(request, paginate).configure(table)
             table_config_form = TableConfigForm(table=table)
+            max_page_size = get_settings_or_config("MAX_PAGE_SIZE")
+            if paginate["per_page"] > max_page_size:
+                messages.warning(
+                    request,
+                    f"Requested `per_page`is too large. No more than {max_page_size} items may be displayed at a time.",
```

Comment:
Let's not use backticks in the UI.

```suggestion
                    f'Requested "per_page" is too large. No more than {max_page_size} items may be displayed at a time.',
```